src/github.rs

```diff
@@ -306,6 +306,7 @@ pub struct Comment {
 #[serde(rename_all = "lowercase")]
 pub enum PullRequestReviewState {
     Approved,
+    #[serde(rename(serialize = "changes_requested"))]
```

Comment:
I think this can be done by changing the `rename_all` above to `snake_case`?

Reply:
good point, thanks